CHANGELOG.md

```diff
@@ -1,6 +1,39 @@
 # Change Log
 
-## [v0.3.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.3.0) (2020-06-01)
+## [0.4.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.4.0) (August 10, 2020)
+[Full Changelog](https://github.com/mongodb/go-client-mongodb-atlas/compare/v0.3.0...0.4.0)
+
+**Implemented enhancements:**
+
+- Organizations endpoints [\#100](https://github.com/mongodb/go-client-mongodb-atlas/issues/100)
```

Comment:
This script used doesn't really highlight the bigger enhancements unless they have an issue and since our work is tracked elsewhere we need to manually edit.  Add the following to the single line here now.

- Updated Database user for AWS IAM [\#114](https://github.com/mongodb/go-client-mongodb-atlas/pull/114) ([coderGo93](https://github.com/coderGo93))
- Added Custom AWS DNS support [\#112](https://github.com/mongodb/go-client-mongodb-atlas/pull/112) ([gssbzn](https://github.com/gssbzn))
- Implemented Atlas Search support [\#111](https://github.com/mongodb/go-client-mongodb-atlas/pull/111) ([gssbzn](https://github.com/gssbzn))
- Added Atlas Data Lakes support [\#108](https://github.com/mongodb/go-client-mongodb-atlas/pull/108) ([robcarlan-mlab](https://github.com/robcarlan-mlab))
- Added Online Archive support [\#107](https://github.com/mongodb/go-client-mongodb-atlas/pull/107) ([gssbzn](https://github.com/gssbzn))
- Added vanity URL [\#87](https://github.com/mongodb/go-client-mongodb-atlas/pull/87) ([gssbzn](https://github.com/gssbzn))

Reply:
Thanks I will add them